### DISCUSSION TITLE
Difference between jaegertracing/all-in-one and jaegertracing/opentelemetry-all-in-one

### DISCUSSION THREAD

**Discussion by hongbo-miao** [open-telemetry/opentelemetry-collector#2558] [Q&A] [answered]
In the [docker-compose.yaml](https://github.com/open-telemetry/opentelemetry-collector/blob/main/examples/demo/docker-compose.yaml) of the demo under this repo, it is using `jaegertracing/all-in-one`.

However, these is another `jaegertracing/opentelemetry-all-in-one` at https://hub.docker.com/r/jaegertracing/opentelemetry-all-in-one/

What are the difference? Which one should we use? Thanks

**Comment by bogdandrutu** [comment]
@jpkrohling do you know how to answer that?

**Reply by bogdandrutu** [reply]
Or maybe @pavolloffay

**Reply by pavolloffay** [reply]
The `jaegertracing/opentelemetry-all-in-one` is an experimental Jaeger build on top of OTEL collector

**Reply by lihaomin1010** [reply]
> The `jaegertracing/opentelemetry-all-in-one` is an experimental Jaeger build on top of OTEL collector

Could you tell me how to build opentelemetry-all-in-one?

**Comment by jpkrohling** [comment, marked as answer]
You should be using the `jaegertracing/all-in-one`, as we just discontinued the other one. Jaeger v2 might be based on the OpenTelemetry Collector, built with the lessons learned from the image we discontinued.
